package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "serialize-error",
-	"version": "6.0.0",
+	"version": "7.0.0",
```

Comment:
You should generally not bump the version in a PR. It's up to the maintainers to decide when and what version to use.

Reply:
fixed